$MirOS: ports/multimedia/livestream/patches/patch-liveMedia_MPEG1or2Demux_cpp,v 1.1.7.1 2005/03/18 15:50:47 tg Exp $
--- liveMedia/MPEG1or2Demux.cpp.orig	Thu May 12 01:40:47 2005
+++ liveMedia/MPEG1or2Demux.cpp	Thu May 19 19:58:58 2005
@@ -21,6 +21,7 @@ along with this library; if not, write t
 #include "MPEG1or2Demux.hh"
 #include "MPEG1or2DemuxedElementaryStream.hh"
 #include "StreamParser.hh"
+#include <stddef.h>
 #include <stdlib.h>
 
 ////////// MPEGProgramStreamParser definition //////////
@@ -161,7 +162,7 @@ void MPEG1or2Demux::registerReadInterest
   // Make sure this stream is not already being read:
   if (out.isCurrentlyAwaitingData) {
     envir() << "MPEG1or2Demux::registerReadInterest(): attempt to read stream id "
-	    << (void*)streamIdTag << " more than once!\n";
+	    << (void*)(ptrdiff_t)streamIdTag << " more than once!\n";
     exit(1);
   }
     
@@ -456,7 +457,7 @@ void MPEGProgramStreamParser::parsePackH
     skipBytes(pack_stuffing_length);
   } else { // unknown
     fUsingSource->envir() << "StreamParser::parsePack() saw strange byte "
-			  << (void*)nextByte
+			  << (void*)(ptrdiff_t)nextByte
 			  << " following pack_start_code\n";
   }
   
